framework/PostProcessors/BasicStatistics.py

```diff
@@ -105,6 +105,9 @@ class cls.
     biasedInput = InputData.parameterInputFactory("biased", contentType=InputData.StringType)
     inputSpecification.addSub(biasedInput)
 
+    multipleLinearInput = InputData.parameterInputFactory("multipleLinear", contentType=InputData.StringType)
```

Comment:
The type should be boolean type and not string.

Reply:
fixed